.travis.yml

```diff
@@ -1,12 +1,27 @@
-git:
-  depth: 10
+language: node_js
+node_js:
+- '0.11'
+- '0.12'
+- '4.0'
+- '5.0'
+- '6.0'
+os:
+- linux
+- osx
+script: curl -s https://raw.githubusercontent.com/atom/ci/master/build-package.sh
```

Comment:
Unless you use the following script (or one similarly modified), all the Node.js changes requested above will be ignored, since it will use the `apm` Node.js (v0.10.40):

```text
https://raw.githubusercontent.com/Arcanemagus/ci/atomlinter/build-package.sh
```